config/feature_flags.yml

```diff
@@ -10,10 +10,15 @@ feature_flags:
     uat: false
     production: false
   insights:
-    local: <%= ENV.fetch("INSIGHTS", false) %>
+    local: <%= ENV.fetch("INSIGHTS_ENABLED", false) %>
     development: true
     uat: false
     production: false
+  nsm:
```

Comment:
I don't like the name of this feature flag - NSM absolutely is enabled, and caseworkers can see the NSM area of the app when they log in. Can it be `nsm_insights`?